android/src/test/java/com/thebluealliance/androidclient/test/models/MatchTest.java

```diff
@@ -0,0 +1,77 @@
+package com.thebluealliance.androidclient.test.models;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.thebluealliance.androidclient.models.Match;
+
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Jerry on 4/15/2015.
+ */
+@RunWith(RobolectricTestRunner.class)
+public class MatchTest {
+    private static final String ALLIANCES_2009 = "{" +
+            "\"red\":  {\"score\": 81, \"teams\": [\"frc68\", \"frc217\", \"frc247\"]}," +
+            "\"blue\": {\"score\": 98, \"teams\": [\"frc111\", \"frc971\", \"frc67\"]}" +
+            "}";
+    private static final JsonObject ALLIANCES = new JsonParser().parse(ALLIANCES_2009).getAsJsonObject();
```

Comment:
Since JsonObject is mutable, it'd be better to create it in a @Before method. I'll do that now.